packages/firebase_core/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.5.0
+
+* Migrate To AndroidX.
+* This is a breaking change and requires manual migration. [Click here for more details](https://flutter.dev/docs/development/packages-and-plugins/androidx-compatibility#how-to-migrate-a-flutter-app-to-androidx)
```

Comment:
Can you clarify what situations are being improved by this change? I don't think we want to force Flutterfire users to manually migrate their apps.

I'm wondering if landing https://github.com/FirebaseExtended/flutterfire/pull/39 would be sufficient for your use case.